src/algorithm/broad_phase/sweep_prune.rs

```diff
@@ -83,11 +82,11 @@ where
         }
 
         shapes.sort_by(|a, b| {
-            let cmp_min = a.get_bound().min()[self.sweep_axis]
-                .partial_cmp(&b.get_bound().min()[self.sweep_axis]);
+            let cmp_min = a.get_bound().min_extent()[self.sweep_axis]
```

Comment:
let's rename `get_bound()` -> `bound()`, which highlights the fact it's just an accessor. If it's not, we should only call it once per iteration here.